sdk/src/Core/Amazon.Runtime/Internal/Util/Logger.cs

```diff
@@ -60,14 +58,54 @@ private void ConfigsChanged(object sender, PropertyChangedEventArgs e)
         private void ConfigureLoggers()
         {
             LoggingOptions logging = AWSConfigs.LoggingConfig.LogTo;
+            bool found = true;
             foreach (InternalLogger il in loggers)
             {
-                if (il is InternalLog4netLogger)
-                    il.IsEnabled = (logging & LoggingOptions.Log4Net) == LoggingOptions.Log4Net;
-                if (il is InternalConsoleLogger)
-                    il.IsEnabled = (logging & LoggingOptions.Console) == LoggingOptions.Console;
-                if (il is InternalSystemDiagnosticsLogger)
-                    il.IsEnabled = (logging & LoggingOptions.SystemDiagnostics) == LoggingOptions.SystemDiagnostics;
+                if (il is InternalLog4netLogger && (logging & LoggingOptions.Log4Net) == LoggingOptions.Log4Net)
+                {
+                    il.IsEnabled = true;
```

Comment:
With this code `IsEnabled` is always being set to true but the this method is called if the logging configuration changes which might disable a logger.